python/ray/rllib/agents/ppo/appo_policy.py

```diff
@@ -276,8 +394,42 @@ def add_values_and_logits(policy):
     return out
 
 
-AsyncPPOTFPolicy = VTraceTFPolicy.with_updates(
+class TargetNetworkMixin(object):
```

Comment:
Can you add some comments on why theoretically this makes sense (and links to relevant papers?)

Reply:
There are no relevant papers; all workers importance sample themselves to a target network, which is updated by master learner every X iterations. This makes sure that pi_old is stable.